### PR TITLE
Display indico version in juju status

### LINKED ISSUE #138: We should display the deployed version of Indico in juju application status
### Enhancement Proposal

If you deploy indico currently you get the following as part of `juju status` output:
```
App     Version  Status   Scale  Charm   Channel  Rev  Address         Exposed  Message
indico           waiting    0/1  indico  stable    73  10.152.183.192  no       installing agent
```
We should also include the version of Indico that's running in the relevant field above.

### PR DESCRIPTION
**Pull request by arturo-seijas**
Closes #138

### CODE REVIEW

File: src/charm.py [hunk 1]
```diff
@@ -471,6 +473,13 @@ def _install_plugins(self, container, plugins):
             )
             process.wait_output()
 
+    def _get_indico_version(self):
+        container = self.unit.get_container("indico")
+        process = container.exec(["indico", "--version"])
+        version_string, _ = process.wait_output()
+        version = findall(r"[0-9.]+", version_string)
```
Comment (weiiwang01) [inline src/charm.py:480]: A little picky comment, I think raw string is unnecessary here.

**Review by weiiwang01 — DISMISSED** [review]

File: src/charm.py (hunk 1 above)
Comment (arturo-seijas) [reply src/charm.py:480]: You're right

File: src/charm.py [hunk 2]
```diff
@@ -471,6 +473,13 @@ def _install_plugins(self, container, plugins):
             )
             process.wait_output()
 
+    def _get_indico_version(self):
+        container = self.unit.get_container("indico")
+        process = container.exec(["indico", "--version"])
```
Comment (mthaddon) [inline src/charm.py:478]: Can we use a full path here?

Comment (arturo-seijas) [reply src/charm.py:478]: We can

**Review by mthaddon — APPROVED** [review]
LGTM, thx

**Review by jdkandersson — APPROVED** [review]

**Pull request merged**